lib/livebook_web/live/form_helpers.ex

```diff
@@ -55,6 +54,38 @@ defmodule LivebookWeb.FormHelpers do
     """
   end
 
+  @doc """
+  Emoji input.
+  """
+  def emoji_input(assigns) do
+    ~H"""
+    <div id={@id} class="flex border-[1px] bg-gray-50 rounded-lg space-x-4 items-center">
+      <div class="grid grid-cols-1 md:grid-cols-3 w-full">
+        <div class="place-content-start">
+          <div class="p-1 pl-3">
+            <span id="emoji-preview"><%= input_value(@form, @field) %></span>
```

Comment:
Need to namespace this and the button too. Since those are for the hook only, you can put a `data-emoji-preview` attribute on it and then `this.el.querySelector("[data-emoji-preview]")`.